types/simple-lru/tslint.json

```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
```

Comment:
For new packages, the file should only `"extends": "dtslint/dt.json"`

Reply:
I used from other folders.
Np, I'll fix.